lib/rules/no-invalid-html-attribute.js

```diff
@@ -208,14 +222,16 @@ const COMPONENT_ATTRIBUTE_MAP = new Map();
 COMPONENT_ATTRIBUTE_MAP.set('rel', new Set(['link', 'a', 'area', 'form']));
 
 const messages = {
-  onlyStrings: '“{{attributeName}}” attribute only supports strings.',
-  noEmpty: 'An empty “{{attributeName}}” attribute is meaningless.',
+  emptyIsMeaningless: 'An empty “{{attributeName}}” attribute is meaningless.',
   neverValid: '“{{reportingValue}}” is never a valid “{{attributeName}}” attribute value.',
-  notValidFor: '“{{reportingValue}}” is not a valid “{{attributeName}}” attribute value for <{{elementName}}>.',
-  spaceDelimited: '”{{attributeName}}“ attribute values should be space delimited.',
+  noEmpty: 'An empty “{{attributeName}}” attribute is meaningless.',
   noMethod: 'The ”{{attributeName}}“ attribute cannot be a method.',
+  notAlone: '“{{reportingValue}}” must be directly followed by “{{missingValue}}”.',
+  notPaired: '“{{reportingValue}}” can not be directly followed by “{{secondValue}}” without “{{missingValue}}”.',
+  notValidFor: '“{{reportingValue}}” is not a valid “{{attributeName}}” attribute value for <{{elementName}}>.',
```

Comment:
also this - let's revert all unrelated formatting/sorting

Reply:
Sure 67b05f7